Godeps/Godeps.json

```diff
@@ -447,6 +447,11 @@
 			"Rev": "9ef341037b1bcadeeee79e77da4f6baf63de4feb"
 		},
 		{
+			"ImportPath": "github.com/shirou/gopsutil/load",
+			"Comment": "v2.0.0-10-g9ef3410",
+			"Rev": "9ef341037b1bcadeeee79e77da4f6baf63de4feb"
```

Comment:
That's the version of gopsutil already packaged with rkt. Looking now, it appears that gopsutil does have tagged releases, so we should switch to using that. If @woodbor doesn't want to make the switch in this PR I can put together a separate one to switch the version used.